apps/system/test/apps/wifi-direct/js/main.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+(function() {
+  var list = document.querySelector('section ul');
+  var listView = new window.WifiP2pPeerListView(list);
```

Comment:
Since 'WifiP2pPeerListView' is explicitly exported as window's property in wifip2p_listview.js, don't you think explicitly specifying 'window.WifiP2pPeerListView' is better?